core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -111,7 +111,7 @@ <h3 class="topic-card-header">Details</h3>
               <textarea type="text"
                         class="form-control topic-description e2e-test-topic-page-title-fragment-field"
                         [(ngModel)]="editablePageTitleFragmentForWeb"
-                        [maxlength]="MAX_CHARS_IN_PAGE_TITLE_FRAGMENT_FOR_WEB"
+                        [maxlength]="maxWebTitleFrag"
```

Comment:
Please don't abbreviate words, and choose your variable names more carefully -- I don't know what a "maxWebTitleFrag" is.

With max/min there needs to be some idea of what the max/min is being applied to (in this case, characters) but that's not clear from the variable name. Please revise, here and elsewhere.